packages/nextjs/src/config/loaders/dataFetchersLoader.ts

```diff
@@ -111,6 +111,21 @@ export default function wrapDataFetchersLoader(this: LoaderThis<LoaderOptions>,
   // We know one or the other will be defined, depending on the version of webpack being used
   const { projectDir, pagesDir } = 'getOptions' in this ? this.getOptions() : this.query;
 
+  // Get the parameterized route name from this page's filepath
+  const parameterizedRouteName = path
```

Comment:
Nice chaining approach! I like it! (I wish it'd work as nicely in all our SDKs)

Reply:
Props go to Katie.